asr1k_neutron_l3/models/neutron/l3/interface.py

```diff
@@ -51,49 +52,115 @@ def all_interfaces(self):
 
         return result + self.internal_interfaces + self.orphaned_interfaces
 
+    def get_internal_networks(self, ip_version):
+        cidrs = []
+        for interface in self.internal_interfaces:
+            for subnet in interface.subnets:
+                if subnet.get('cidr') and utils.get_ip_version(subnet['cidr']) == ip_version:
+                    cidrs.append(subnet['cidr'])
+        cidrs.sort()
```

Comment:
out of curiosity, do we need to have this sorted for any API or is this for the readability of a human being?

Reply:
I sort these lists, as order is not guaranteed by OpenStack and is in fact considered to be unordered, but when this is the source for a prefix list or interface configuration, order might suddenly matter. Most classes do a sort themselves, but just on the off-chance that we missed it somewhere I like to keep it sorted.

In the past we had some prefix lists that changed their order on each update, causing a lot of config changes for "nothing has changed".

Reply:
whee thanks for the details